docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -0,0 +1,99 @@
+# Using Visual Studio to Develop Enclave Applications for Windows
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Windows.  If you instead want to use Visual
+Studio Code, see the
+[VS Code instructions](https://github.com/openenclave/openenclave/blob/master/devex/vscode-extension/README.md).
+
+## Prerequisites
+
+You will need the following:
+
+- A development machine with support for SGX1 or SGX1 with Flexible Launch Control (FLC) (see [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) for how to
+determine this).
+- [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/)
+  (Community edition, or any other edition) with the "Desktop development
+  with C++" workload
+- NuGet Package Manager feature, installable via Tools -> Get Tools and Features... ->
+  Individual components -> Code tools -> NuGet Package Manager (this should
+  be already installed by the previous step above)
+- [Open Enclave Wizard - Preview](https://marketplace.visualstudio.com/items?itemName=MS-TCPS.OpenEnclaveSDK-VSIX)
+  Visual Studio extension, v0.7 or later.  The extension can be installed via that marketplace link, or from within
+  Visual Studio.  (Do Extensions -> Manage Extensions -> Online -> search for "enclave".)  You must restart Visual Studio after
+  installing the extension. [HACKATHON private](https://1drv.ms/u/s!Aqj-Bj9PNivcnu9rlOlmiAVZz-jOtg?e=QlcO7t)
+- Intel's PSW2.6. This should be automatically installed on Windows 10 systems that support SGX.  If manual install is needed:
+  - For systems with support for SGX1: [Intel's PSW 2.6,Intel Enclave Common API library](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md)
+  - For systems with support for SGX1+FLC: [Intel's PSW2.6, Intel's Data Center Attestation Primitives and related dependencies](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md)
+- [Clang/LLVM for Windows 64-bit](http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe)
```

Comment:
The installer asks whether LLVM should be added to the system PATH. It seems like the answer is no, but I'm not sure. Could you add a note with the right answer?

Reply:
Looks like the answer is to add LLVM to the PATH for at least the current user. Installing LLVM without modifying the PATH accordingly results in VS not being able to find clang.

Reply:
Added note as requested